introduction.md

```diff
@@ -1 +1,4 @@
-This is some introduction text. The best CodeSchool track was iOS.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?